Gemfile

```diff
@@ -19,7 +19,7 @@ group :development, :test do
   # auto-load factories from spec/factories
   gem 'factory_bot_rails'
 
-  gem 'rails'
+  gem 'rails', '~>  7.0'
```

Comment:
We were getting bundle install timeouts; pinning to rails 7.0 is the solution https://github.com/rubygems/rubygems/issues/4539